tests/Cache/CacheFileStoreTest.php

```diff
@@ -11,20 +11,6 @@
 
 class CacheFileStoreTest extends TestCase
 {
-    protected function setUp(): void
-    {
```

Comment:
We never need to pretent that Carbon::now is different than now in this test class

Reply:
@lucasmichot https://github.com/laravel/framework/runs/5798854285?check_suite_focus=true

This causes `testIncrementDoesNotExtendCacheLife()` to become brittle, intermittently failing if the second ticks over in the middle of this test case. Test setup `Carbon::now()->addSeconds(50)->getTimestamp()` and `FileStore@put()` -> `InteractsWithTime@availableAt()` code `Carbon::now()->addRealSeconds($delay)->getTimestamp()` must originate from the exact same `Carbon::setTestNow()` marker. When it doesn't, hash generation in the cache service can mismatch the setup test mock.

This brings into question every other `Carbon::setTestNow(Carbon::now())` removal in this PR.